pkg/api/testing/unstructured_test.go

```diff
@@ -167,9 +171,16 @@ func TestRoundtripToUnstructured(t *testing.T) {
 			{Group: "resource.k8s.io", Version: "v1alpha2", Kind: "ResourceClaimParametersList"},
 			{Group: "resource.k8s.io", Version: "v1alpha2", Kind: "ResourceClassParameters"},
 			{Group: "resource.k8s.io", Version: "v1alpha2", Kind: "ResourceClassParametersList"},
+			{Group: "apiextensions.k8s.io", Version: "v1beta1", Kind: "ConversionReview"},
+			{Group: "apiextensions.k8s.io", Version: "v1", Kind: "ConversionReview"},
 		},
 	}
 
+	typesWithNoInternalVersions := []schema.GroupVersionKind{
```

Comment:
Nit: This could be a `map[GroupVersionKind]struct{}` since it's only being used for a membership test, instead of needing to scan the slice.